stdlib/3/builtins.pyi

```diff
@@ -610,7 +610,6 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def __init__(self, iterable: Iterable[Tuple[_KT, _VT]], **kwargs: _VT) -> None: ...
     def clear(self) -> None: ...
     def copy(self) -> Dict[_KT, _VT]: ...
-    def pop(self, k: _KT, default: _VT = None) -> _VT: ...
```

Comment:
Seems a bit arbitrary to remove pop but to leave `clear()`, etc. We should prune more methods in a follow-up diff.